torch/nn/utils/_expanded_weights/conv_expanded_weights.py

```diff
@@ -17,6 +17,10 @@ def forward(ctx, kwarg_names, conv_fn, *expanded_args_and_kwargs):
         expanded_args, expanded_kwargs = conv_args_and_kwargs(kwarg_names, expanded_args_and_kwargs)
         output = forward_helper(conv_fn, expanded_args, expanded_kwargs)
         input, weight = expanded_args
+        batched_dim_size = conv_picker(conv_fn, 3, 4, 5)
+        if input.dim() != batched_dim_size:
+            raise RuntimeError(f"Expanded Weights only support convolution with batched input, got {conv_fn} with an"
+                               f"unbatched input of dim {input.dim()}, expected input of dim {batched_dim_size}")
```

Comment:
Add a manual test for this error message maybe? Just to be sure this gets triggered?